daemon/containerd/image_manifest.go

```diff
@@ -48,6 +49,51 @@ func (i *ImageService) walkImageManifests(ctx context.Context, img containerdima
 	return errNotManifestOrIndex
 }
 
+// walkReachableImageManifests calls the handler for each manifest in the
+// multiplatform image that can be reached from the given image.
+// The image might not be present locally, but its descriptor is known.
+// The image implements the containerd.Image interface, but all operations act
```

Comment:
The image you are talking about is the `img` parameter right? That implements `images.Image`? I'm confused, I don't see any [containerd.Image](https://pkg.go.dev/github.com/containerd/containerd#Image) here

Reply:
It was meant to describe the `img ImageManifest` parameter which is an implementation of the [containerd.Image](https://github.com/containerd/containerd/blob/926c9586fe4a6236699318391cd44976a98e31f1/image.go#L43) interface.

But I think it doesn't really make sense to include here.. Moved this comment to the `ImageManifest` struct Godoc.